### PR TITLE
Feature/paginate

### PR DESCRIPTION
**Pull request by s-r-x**
Pagination ^-^
#114 

- Pass _start/_end query params to adapters inside router
- minor state redesign for a new "page" variable and reset it on every "selectedStatuses" update
- view powered by react-paginate

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,11 +49,12 @@
     "date-fns": "2.10.0",
     "ejs": "3.0.1",
     "express": "4.17.1",
-    "highlight.js": "^10.4.1",
+    "highlight.js": "^9.18.3",
```
Comment (vcapretz) [inline package.json:52]: do this one needs to be rebased with master so we don't downgrade the version?

Comment (s-r-x) [reply package.json:52]: That specific commit(75ddcfd) can be removed actually, if highlight.js error was fixed on master

Comment (s-r-x) [reply package.json]: rebased to master

**Comment by MatayoshiMariano** [comment]
@s-r-x why this was deleted? bull-board is not showing all jobs

**Pull request closed without merging**